From 8df4a8e32596e554e043ba5bdb11f9fe5e094546 Mon Sep 17 00:00:00 2001
From: Binrui Dong <brett.browning.dong@gmail.com>
Date: Mon, 4 Oct 2021 17:05:17 +0800
Subject: [PATCH] Rollback FreeType to 2.10.4

---
 src/freetype.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/freetype.mk b/src/freetype.mk
index bfa55337..ff8d3cdb 100644
--- a/src/freetype.mk
+++ b/src/freetype.mk
@@ -3,8 +3,8 @@
 PKG             := freetype
 $(PKG)_WEBSITE  := https://www.freetype.org/
 $(PKG)_IGNORE   :=
-$(PKG)_VERSION  := 2.11.0
-$(PKG)_CHECKSUM := 8bee39bd3968c4804b70614a0a3ad597299ad0e824bc8aad5ce8aaf48067bde7
+$(PKG)_VERSION  := 2.10.4
+$(PKG)_CHECKSUM := 86a854d8905b19698bbc8f23b860bc104246ce4854dcea8e3b0fb21284f75784
 $(PKG)_SUBDIR   := freetype-$($(PKG)_VERSION)
 $(PKG)_FILE     := freetype-$($(PKG)_VERSION).tar.xz
 $(PKG)_URL      := https://$(SOURCEFORGE_MIRROR)/project/freetype/freetype2/$(shell echo '$($(PKG)_VERSION)' | cut -d . -f 1,2,3)/$($(PKG)_FILE)
-- 
2.30.1 (Apple Git-130)

